src/components/LinearProgressBar/LinearProgressBar.tsx

```diff
@@ -32,18 +31,24 @@ const LinearProgressBar: React.FC<LinearProgressBarProps> = (props) => {
 	return (
 		<>
 			{props.value > 0 ? (
-				<div className={`relative w-100 h-3 br-10 overflow-x-hidden ${props.className}`} style={props.style}>
-					<div className={`absolute w-100 h-3 br-10 bg-43`} style={{ background: `${props.backgroundColor}` }} />
+				<div className={`relative h-[.75rem] w-full overflow-x-hidden rounded-xl ${props.className}`} style={props.style}>
+					<div className={`absolute h-[.75rem] w-full rounded-xl bg-gray-600`} style={{ background: `${props.backgroundColor}` }} />
 					<div
-						className={`subline absolute h-3 br-10 bg-teal100`}
+						className={`absolute h-[.75rem] rounded-xl bg-[#1da0bc] transition-width`}
```

Comment:
@manishdoley23 here as well, use h-7, and remove 1da0bc color, add the blue-600 color here